Business resume homepage/index.html

```diff
@@ -0,0 +1,107 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="homepage.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.0/css/all.min.css" integrity="sha512-xh6O/CkQoPOWDdYTDqeRdPCVd1SpvCA9XXcUnZS2FmJNp1coAFzvtCN9BmamE+4aHK8yyUHUSCcJHgXloTyT2A==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <title>Business Resume</title>
+</head>
+<body>
+    <header>
+        <div class="header-left">
+            <div class="header-box">
+                <div class="square">
+                </div>
+                <a href="index.html">Maya Nelson</a>
+            </div>
+            <span>/</span>
+            <a class="title-header" href="index.html">PROJECT MANAGER</a>
+        </div>
+
+        <div class="header-right">
+            <input id="toggle" type="checkbox"></input>
```

Comment:
On the MDN page for input type checkbox, is there a closing tag?https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/checkbox

Reply:
done